source/partials/next_release/_release.md.erb

```diff
@@ -13,25 +12,22 @@ This improves performance for large GoCD instances with thousands of pipelines.
 a javascript based frontend that is several thousand times faster on large instances, reducing the CPU usage of your browser. Your laptop battery will thank you. :)
 * reduces the polling rate when the dashboard is shown in a browser tab that is hidden.
 
-> Please report any issues that you observe on [GitHub issues](https://github.com/gocd/gocd/issues).
 
-
-<h4>Plugin Infrastructure Enhancements</h4>
-We have been working on a few improvements to our plugin infrastructure.
+<h4>Plugin Enhancements</h4>
+We have been working on a few improvements to our plugin architecture.
 
 **Errors and Warnings support:**
 We have a [new API](https://plugin-api.gocd.org/18.3.0/notifications/#add-server-health-messages) that allows plugins to add Errors and Warnings to the Go Server. These will show up along with the server's own Errors and Warnings. It will allow plugin developers to notify users if any errors or exceptions occur.
```

Comment:
Replace this line with: "We have a [new API](https://plugin-api.gocd.org/18.3.0/notifications/#add-server-health-messages) which allows plugin developers to notify users if any errors or exceptions occur. Using this, plugin developers can add errors and warnings to the GoCD Server. These will show up along with the applications's own warnings on the 'Errors and Warnings' section on the GoCD Dashboard."